src/screens/Customers/CustomerProfile.tsx

```diff
@@ -162,6 +163,7 @@ function CustomerProfile(props: CustomerProps) {
 
 const mapStateToProps = (state: RootState) => ({
   currentSite: state.siteData.currentSite || EMPTY_SITE,
+  customer: getCurrentCustomer(),
```

Comment:
I'm also down to just do `state.customerData.currentCustomer` and removing the getter method entirely. I initially wanted to do the `getter` in the case that our `state.customerData.siteToCustomers` was in the form of `Record<SiteId, Record<CustomerId, CustomerRecord>>` instead of `Record<SiteId, CustomerRecord[]>`

Reply:
Yeah, I think that makes more sense - it's better to be consistent (so match the above line if possible). You can always refactor later if necessary

Reply:
Ok yeah, good point. I'll change this to be consistent and get rid of `getCurrentCustomer` for now since this was its only use.